INSTALL.md

```diff
@@ -46,6 +46,16 @@ If you have an existing Idris 2, go to Step 3. Otherwise, read on...
 Make sure that:
 
 - `$PREFIX/bin` is in your `PATH`
+- `$PREFIX/lib` is in your `LD_LIBRARY_PATH` or `DYLD_LIBRARY_PATH` if on
+  `macOS` (so that the system knows where to look for library support code)
```

Comment:
This was not necessary for me when I bootstrapped or built from previous install on `aarch64-darwin`. It's possible this situation was made better by recent changes to the build scripts, but it's also possible that you found  this change necessary because of a different in my machine's setup and yours or a different in how we were checking whether things were working.

I'd be interested to learn more if you have any recollection after so long!